bundle/edu.gemini.spdb.rollover.servlet/src/main/scala/edu/gemini/rollover/servlet/RolloverServlet.scala

```diff
@@ -55,10 +55,12 @@ final class RolloverServlet(site: Site, odb: IDBDatabaseService, user: java.util
     <obs>
       <id>{ ro.id.toString }</id>
       <partner>{ ro.partner }</partner>
-      <target>
-        <ra>{ ro.target.coords.map(_.ra).getOrElse(0.0) }</ra>
-        <dec>{ ro.target.coords.map(_.dec).getOrElse(0.0) }</dec>
-      </target>
+      {ro.targets.map { t =>
+        <target>
+          <ra>{  t.coords.map(_.ra).getOrElse(0.0) }</ra>
+          <dec>{ t.coords.map(_.dec).getOrElse(0.0) }</dec>
+        </target>
+      }}
```

Comment:
Ok, that's helpful. Initially I guess ITAC can just look at the first target (that's probably what will happen anyway if it's just selecting the element by path). But I will check.

Re: merging this isn't going to happen any time soon so you don't need to worry about upcoming ITAC.